Danger-Swift/Package.resolved

```diff
@@ -1,6 +1,24 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "DangerSwiftCoverage",
+        "repositoryURL": "https://github.com/f-meloni/danger-swift-coverage",
+        "state": {
+          "branch": null,
```

Comment:
What does this default to? Master? 
Or does it work though the revision which is a commit id?

Reply:
It uses the tag!